### ISSUE TITLE
What does this do?

### ISSUE THREAD

**Issue by Visual-Studio-Coder** [33tm/Gradar#1]
Hi 33tm. I noticed that it lets me sign in, which is important, but what does it do after that?

**Comment by Visual-Studio-Coder** [comment]
Like, it just redirects me to the starting page instead of showing me classes or other data

**Comment by 33tm** [comment]
Oh interesting, looks like the redirect to `/app` fails on the production build...

It's meant to be a grade calculator (although quite unfinished), thanks for pointing the issue out!

**Comment by Visual-Studio-Coder** [comment]
Yeah I noticed that on the callback URL thing too! I would be so stoked to use this. I have never been comfortable with this other service "Bessy"s privacy practices

**Comment by Visual-Studio-Coder** [comment]
I noticed that when I try to sign in, it also says 
![image](https://github.com/33tm/Gradar/assets/78756662/21b5818f-dddf-4803-ad52-596cfa5c5f23)
but there IS a way to get it to say the actual application name instead to seem more legit.

If you have a dedicated developer key from a schoology developer account, it works, and I can generate one for you if you would like.

**Comment by 33tm** [comment]
I thought they disappeared completely with the PowerSchool rebranding, how'd you manage to get the dedicated name with a developer account? The current credentials are from a developer account.

**Comment by Visual-Studio-Coder** [comment]
Oh. You create a new application, and they give you keys for the application

And you can create a name for the application, and this is what shows up in the oauth page

**Comment by 33tm** [comment]
That works beautifully, thank you so much for letting me know!

**Comment by Visual-Studio-Coder** [comment]
I get that you're busy but please do update this soon. I'd really enjoy an open source grade checker.

**Comment by 33tm** [comment]
Will try to get an update out soon, glad to hear the potential it has 😅

**Comment by 33tm** [comment]
@Visual-Studio-Coder Just created [One](https://gunn.one), it's not open-sourced just yet but hope it works for you :) apologies for the long delay
